src/server.ts

```diff
@@ -1,23 +1,78 @@
 import { AxiosResponse } from "axios";
-import express, { Express } from "express";
+import express from "express";
  
 const expressApp = express();
 const port = 8000;
-const axios = require('axios');
+const axios = require("axios");
+const apiKey = "IqA8fYkIhjXf62lkAuWM3WqexG4waX0mbovOUeII"
+enum Camera {
+    Fhaz = "fhaz",
+    Rhaz = "rhaz",
+    Mast = "mast",
+    Chemcam = "chemcam",
+    Mahli = "mahli",
+    Mardi = "mardi",
+    Navcam = "navcam",
+    Pancam = "pancam",
+    Minites = "minites",
+}
+
+enum Rover {
+    Curiosity = "curiosity",
+    Spirit = "spirit",
+    Opportunity = "opportunity",
+    Perseverance = "perseverance",
+}
```

Comment:
There are generally two places to put things like enums:
* If it's used in only one file, put it in that file
* Otherwise, a sort of constants file (or its own, if it makes sense)